tests/installation/installation_overview.pm

```diff
@@ -42,6 +42,9 @@ sub ensure_ssh_unblocked {
                 send_key_until_needlematch 'ssh-blocked-selected', 'tab', 25;
                 send_key 'ret';
                 send_key_until_needlematch 'ssh-open', 'tab';
+                send_key_until_needlematch 'ssh-service-disabled', 'tab', 25;
+                send_key 'ret';
+		send_key_until_needlematch 'ssh-service-enabled', 'tab';
```

Comment:
Hi Sarah,

This can cause false positives, since the enabling is happening unconditionally. There are some cases where the expected default are tested and installation_overview is executed in any interactive installation.
I can think of adding a job setting like `ENABLE_SSHD_BEFORE_INSTALL=1` and then

```suggestion
                if (check_var('ENABLE_SSHD_BEFORE_INSTALL', 1) {
                  send_key_until_needlematch 'ssh-service-disabled', 'tab', 25;
                  send_key 'ret';
                }
```

Reply:
ah, sure, I missed a parenthesis :facepalm: 

```suggestion
                if (check_var('ENABLE_SSHD_BEFORE_INSTALL', 1)) {
                  send_key_until_needlematch 'ssh-service-disabled', 'tab', 25;
                  send_key 'ret';
                  send_key_until_needlematch 'ssh-service-enabled', 'tab';
                }
```